### DISCUSSION TITLE
Option to provide custom functions to deploy alongside the 'ssr' function

### DISCUSSION THREAD

**Discussion by eburi** [angular/angularfire#2607] [Ideas]
# Version info

**Angular:**  10.0.11

**Firebase:** 7.13.1

**AngularFire:** 6.0.3

**Other (e.g. Ionic/Cordova, Node, browser, operating system):**

### How to reproduce these conditions

**Steps to set up and reproduce**

New Angular Project with universal and angular/fire

### Expected behavior

angularfire should provide an option to specify our own functions that will be included in the deployed packages. 

### Actual behavior

https://github.com/angular/angularfire/blob/master/src/schematics/deploy/functions-templates.ts 
Generates a custom project with one default-Function 'ssr' and a default 'package.json'.

**Comment by Proibito04** [comment]
any updates?

**Comment by MikeG1995** [comment]
same problem :(

**Comment by wjvantilburg** [comment]
How did you guys tackle this?
